ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js

```diff
@@ -209,38 +209,66 @@ export default {
       response: Responses.goRight(),
     },
     {
-      // TODO: use headmarkers for this
+      id: 'EmeraldEx Emerald Crusher',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5585', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '5585', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '5585', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '5585', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'EmeraldEx Secundus Terminus est Corner',
+      netRegex: NetRegexes.ability({ source: 'Bitblade', id: '55C9', capture: false }),
+      alarmText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go to intercards',
+          cn: '去场边四角',
+        },
+      },
+    },
+    {
       id: 'EmeraldEx Secundus Terminus est',
-      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55C8', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '55C8', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '55C8', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '55C8', capture: false }),
+      netRegex: NetRegexes.ability({ source: 'Bitblade', id: '55CA', capture: false }),
       alarmText: (data, _, output) => output.text(),
       outputStrings: {
         text: {
-          en: 'X to cards, + to intercards',
-          de: 'X in die Cardinalen, + in die Intercardinale Himmelsrichtungen',
-          cn: '靠边放剑(十字四角)',
-          ko: 'X는 동서남북, +는 대각위치로',
+          en: 'Go to cards',
+          cn: '去场边中间',
         },
       },
     },
     {
-      id: 'EmeraldEx Legio Phantasmatis',
-      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55B4', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '55B4', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '55B4', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '55B4', capture: false }),
+      id: 'EmeraldEx Expire P2',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '282b', capture: false }),
```

Comment:
I think this id is `55D1`? It definitely won't be `282b` though.

Reply:
yep, mb i was looking at the wrong line. Just fixed.